### DISCUSSION TITLE
Welcome to Gitcommands Discussions!

### DISCUSSION THREAD

**Discussion by narendrapb** [Auroprobatch3/Gitcommands#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sravangarla** [comment]
@narendrapujanaboyina  Broo try to perform squash once it's not working in my system.

**Reply by narendrapb** [reply]
we will see
